.github/workflows/pdf.yml

```diff
@@ -0,0 +1,80 @@
+name: Build PDF of Docs
+
+on:
+  push:
+    branches: [master]
+    tags:
+      - 'v*'
+  pull_request:
+
+jobs:
+
+  pdfhtml:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install wheel
+        pip install -e .[sphinx,pdfhtml]
+    - name: Install Headless Chrome dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -yq $(cat .github/workflows/pyppeteer_reqs.txt)
+    - name: Install Chromium
+      run: |
+        pyppeteer-install
+    - name: Build PDF from HTML (Docs)
+      run: |
+        jb build docs --builder pdfhtml -n -W --keep-going
+    - uses: actions/upload-artifact@v2
+      with:
+        name: PDF_HTML
+        path: docs/_build/pdf/book.pdf
+
+  pdflatex:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install wheel
+        pip install -e .[sphinx,pdflatex]
+    - name: Install latex dependencies
+      run: |
+        sudo apt-get -qq update
+        sudo apt-get install -y     \
+          texlive-latex-recommended \
+          texlive-latex-extra       \
+          texlive-fonts-recommended \
+          texlive-fonts-extra       \
+          texlive-xetex             \
+          latexmk                   \
+          xindy
+    - name: Build PDF from LaTeX (Docs)
+      run: |
+        jb build docs --builder pdflatex -n -W --keep-going
+    - uses: actions/upload-artifact@v2
+      with:
+        name: PDF_LATEX
+        path: docs/_build/latex/book.pdf
```

Comment:
could we document for other JB developers how they could preview the artifact if they wish to? (in the contribution section perhaps)

Reply:
Good idea. I'll add this.